integrations/tensorflow/python_projects/iree_tflite/iree/tools/tflite/scripts/iree_import_tflite/__main__.py

```diff
@@ -4,18 +4,65 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-import subprocess
+import argparse
+import inspect
+from pathlib import Path
+import re
 import sys
-
 import iree.tools.tflite
 
 
-def main(args=None):
-  if args is None:
-    args = sys.argv[1:]
-  exe = iree.tools.tflite.get_tool("iree-import-tflite")
-  return subprocess.call(args=[exe] + args)
+def main():
+  parser = argparse.ArgumentParser()
+  parser.add_argument('flatbuffer', help='<TFLite FlatBuffer>')
+  parser.add_argument(
+      '-o',
+      '--output-path',
+      dest='output_path',
+      required=True,
+      help='Path to the mlirbc file name to output.',
+  )
+  parser.add_argument(
+      '--input-array',
+      dest='input_arrays',
+      action='append',
+      help='Input tensor, if different from the default inputs',
+  )
+  parser.add_argument(
+      '--output-array',
+      dest='output_arrays',
+      action='append',
+      help='Output tensor, if different from the default inputs',
+  )
+
+  # Deprecated and unused.  Kept in place so callers of the old tool don't break
+  # when using the new tool.
+  parser.add_argument('--output-format',
+                      dest='_',
+                      required=False,
+                      help=argparse.SUPPRESS)
+  args = parser.parse_args()
+  tflite_to_tosa(
+      flatbuffer=args.flatbuffer,
+      bytecode=args.output_path,
+      ordered_input_arrays=args.input_arrays,
+      ordered_output_arrays=args.output_arrays,
+  )
+
+
+def tflite_to_tosa(
+    flatbuffer,
+    bytecode,
+    use_external_constant=False,
+    ordered_input_arrays=None,
+    ordered_output_arrays=None,
+):
+  from tensorflow.python.pywrap_mlir import experimental_tflite_to_tosa_bytecode
```

Comment:
Can this import not be at the top? I'm not seeing the use case for importing this file and not importing this function